keyboards/prime_r/config.h

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2018 Andrew Heaston
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    qmkbuilder
+#define PRODUCT         keyboard
+#define DESCRIPTION     Keyboard
```

Comment:
Would you mind filling these lines out to represent the keyboard?

Reply:
Done.  Unsure on device_ver, believe it is 1.0

Reply:
@jackhumbert - I read up on the device ID information - I'm not sure the manufacturer has gone through the registration process, but I will reach out to them and see.

If they have not, what would you have me do for the vendor / product ID?

Reply:
You can leave the vendor/product ID as the bogus ones we use right now - we're working on a deeper solution for this :)

Reply:
Awesome. :)